src/models/organizationcenter.model.ts

```diff
@@ -3,4 +3,12 @@ export default interface OrganizationCenter {
   name?: string;
   address?: string;
   organization_id?: string;
-}
\ No newline at end of file
+}
+
+export interface Centers {
```

Comment:
Instead of having a separate `Centers` interface, it would be better to add the `country`, `manager` and `status` properties to the `OrganizationCenter` interface.